src/python/turicreate/toolkits/sound_classifier/sound_classifier.py

```diff
@@ -289,7 +289,7 @@ def create(dataset, target, feature, max_iterations=10,
 
     state = {
         '_class_label_to_id': class_label_to_id,
-        '_custom_classifier': custom_NN,
```

Comment:
We'll also need changes where `SoundClassifier` uses this state, since `state._custom_classifier` is now an instance of `SoundClassifierMXNETModel`, not `mx.nn.Sequential` or whatever